src/main/java/org/owasp/esapi/StringUtilities.java

```diff
@@ -90,7 +90,7 @@ public static boolean contains(StringBuilder input, char c) {
     }
 
     /**
-     * Returns the replace value if the value of test is null, "null", or ""
+     * Returns the replace value if the value of test is null, "null" (case-insensitive), or blank
```

Comment:
While you're changing this, please change 'replace' here to '{@code replace}'. I kept thinking it should be 'replaced' until I noticed it was the name of a formal parameter. Thanks.

Reply:
Resolving this since it was addressed in commit [cbac2c8](https://github.com/ESAPI/esapi-java-legacy/commit/cbac2c8f9ee5a09dbdb09ec97d4fabb2642143cc).